### ISSUE TITLE
OCAT crashes HITMAN

### ISSUE THREAD

**Issue by brunogm0** [GPUOpen-Tools/OCAT#40]
HITMAN 

win10  pro

Crash when enabling Overlay.

**Comment by aurolou** [comment]
I pushed a fix, you can get the build [here](https://ci.appveyor.com/api/buildjobs/k18d1aer2c387lx5/artifacts/OCAT_1.2.0.168.exe).

I noticed, that OCAT also hooks into the launcher, which can cause issues. To prevent these, you can do one of the following:

start the launcher -> (set your options as you like) -> enable the overlay -> start the game via the launcher
or
add Launcher.exe to the blackList.txt.

Hope this helps and the new build fixes your issue :)

**Comment by aurolou** [comment]
Unfortunately, I can't reproduce the crash locally, the fix in version 1.2.0.168 solved the issue for me. So I need some further information from you to figure out the root cause of the problem :)

Do I understand correctly, that you just start OCAT, and without even enabling the overlay, HITMAN crashes on launch, but when OCAT is not running, HITMAN does launch without problems?

And how do you enable the overlay? Do you click on 'Start Overlay' in the Capture tab, or do you use the Advanced tab by starting HITMAN via the OCAT-UI? If you use the 'Start Overlay' in the Capture tab option, when do you turn it on: before you start the Launcher, or after you start the launcher but before you start the actual Hitman game? Do you run Hitman using DX11 or DX12?

Thanks :)

**Comment by brunogm0** [comment]
> Do I understand correctly, that you just start OCAT, and without even enabling the overlay, HITMAN crashes on launch, but when OCAT is not running, HITMAN does launch without problems?
Yes, but with launcher.exe in blacklist it changes to only crash after few seconds in the game.  
![ocat_hitman](https://user-images.githubusercontent.com/1731755/45890526-93ac7900-bd99-11e8-87e4-81d70db7f807.PNG)

Silent crashes with DX11 instead of DX12 (error as in the picture);
OCAT click "Start Overlay on" silent crash launcher at options click; 
OCAT open but overlay off,  allows to change options in launcher.exe;

Usually "click on 'Start Overlay' " but also tested advanced tab.  In this case the error dialog is immediate.
Also, tested advanced without "inject overlay at start",  same instant crash dialog;
If using advanced but selecting Launcher.exe, the error dialog changes to "bad conversion";

If overlay off then game start,  hitman.exe is allocating memory steadly;  (then overlay on) -> the crash dialog happens only at first image print;  The hitman.exe still loads when overlay on after start. 
I have the impression taht after a few tries OCAT behaves differently maybe it stores something. 

Other tests:
Never show overlay in:  hitman, skullgirls or FFXV; 
Showed on Superposition, Quake Champions

ps.: RX460 18.8.1

**Comment by aurolou** [comment]
Thanks for the details! I could locally observe a thrown exception which I think is related to your issue, now I am looking for a way to solve it :)

**Comment by aurolou** [comment]
Hey, sorry for the delay. This [version](https://ci.appveyor.com/api/buildjobs/v9oeptbf5fwk43ec/artifacts/OCAT_1.2.0.182.exe) should work for FFXV, if you launch the game from OCAT (Launch App tab) and add as commandline argument: steam://run/637650

For Hitman, the global hook works for me, if not you can also try the explicit hook (launch the game directly from OCAT) with commandline argument: steam://run/236870

Make sure the steam client is running.

**Comment by rys** [comment]
Did that version fix your issue, @brunogm0?

**Comment by brunogm0** [comment]
Sorry, my SSD failed and so i could not test it.
